frontend/src/components/CreateInvoice.jsx

```diff
@@ -33,68 +111,135 @@ function CreateInvoice() {
       setLoading(true);
       const provider = new BrowserProvider(walletClient);
       const signer = await provider.getSigner();
+
+      console.log("data >>>>>> ", data);
+      console.log("acc :", account.address);
+      // 1. Prepare invoice data
+      const invoicePayload = {
+        amountDue: totalAmountDue,
+        dueDate,
+        issueDate,
+        user: {
+          address: account?.address.toString(),
+          fname: data.userFname,
+          lname: data.userLname,
+          email: data.userEmail,
+          country: data.userCountry,
+          city: data.userCity,
+          postalcode: data.userPostalcode,
+        },
+        client: {
+          address: data.clientAddress,
+          fname: data.clientFname,
+          lname: data.clientLname,
+          email: data.clientEmail,
+          country: data.clientCountry,
+          city: data.clientCity,
+          postalcode: data.clientPostalcode,
+        },
+        items: itemData,
+      };
+
+      const invoiceString = JSON.stringify(invoicePayload);
+
+      // 2. Setup Lit
+      const litNodeClient = new LitNodeClient({
```

Comment:
why are we setting up the lit client on every function call, instead of once, just when the component renders?